R/deprecated.R

```diff
@@ -0,0 +1,8 @@
+#' Deprecated functions
+#'
+#' @param ... Additional arguments passed from the deprecated function to its
+#'   replacement.
+#'
+#' @keywords internal
+#' @name deprecated
+NULL
```

Comment:
I started this file expecting to have a few more deprecated functions, but in the end there were only two: `chat_azure()` and `chat_bedrock()`. I personally like collecting deprecations in one place, but I'm happy to remove this file and document the deprecations in the same pattern as `chat_cortex()`. (Or to bring that deprecation here.)